poi-ooxml/src/test/java/org/apache/poi/ss/tests/util/TestXSSFCellUtil.java

```diff
@@ -19,22 +19,15 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 
 import org.apache.commons.codec.DecoderException;
 import org.apache.commons.codec.binary.Hex;
-import org.apache.poi.ss.usermodel.Cell;
-import org.apache.poi.ss.usermodel.FillPatternType;
-import org.apache.poi.ss.usermodel.IndexedColors;
-import org.apache.poi.ss.usermodel.Row;
-import org.apache.poi.ss.usermodel.Sheet;
-import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.*;
```

Comment:
please don't do this - you should respect the existing code style - avoid wildcard imports